### ISSUE TITLE
Does it still work 

### ISSUE THREAD

**Issue by fyelles** [smarterclayton/openshift-go-cart#29]
Trying to add the cartridge.

Creating application 'mygo' ... Invalid cartridge, error downloading from url 'https://cartreflect-claytondev.rhcloud.com/reflect?github=smarterclayton/openshift-go-cart 

Am i missing something ?

**Comment by ajtam** [comment]
yes, cart still works, you can create an app using the openshift console. There is a Add application button, and search for the go cartridge and follow the instructions given.
https://openshift.redhat.com/app/console/applications

Good luck
